docs/examples/sentiment_analysis/kobert_naver_movie.md

```diff
@@ -0,0 +1,307 @@
+# Sentiment Analysis with KoBERT using Movie Review Data
+# KoBERT를 이용한 네이버 영화 리뷰 감정 분석
+
+
+In this tutorial, we will implement a sentiment analysis of [Naver sentiment movie corpus](https://github.com/e9t/nsmc) with a pretrained Korean BERT (**KoBERT**) provided by GluonNLP's Model Zoo.
+
+이번 예제에서는 GluonNLP의 Model Zoo에서 제공하는 한국어 BERT(**KoBERT**)의 pretrained 모델로 네이버 영화리뷰의 감정 분석(sentiment analysis)을 구현해 보겠습니다.
+
+## Setup
+## 사전 설치
+
+Download and install the files required for this example.
+
+이 예제를 위해 필요한 파일을 다운로드하고 설치합니다.
+
+```{.python .input}
+!pip install tqdm
+!pip install mxnet-cu101
+!pip install gluonnlp pandas
+!pip install sentencepiece
```

Comment:
Hi, I think this part is leading faults to our CI. Is it possible that you do not use such shell language within the python code? If you need to any extra packages that are not included in the current settings, please change this file to add it `setup.py`

Reply:
what about just removing these lines?

Reply:
Is this the setup file you are referring to? https://github.com/dmlc/gluon-nlp/blob/v0.9.x/setup.py
Then, I will upload a new compatible version.

Reply:
> Is this the setup file you are referring to? https://github.com/dmlc/gluon-nlp/blob/v0.9.x/setup.py
> Then, I will upload a new compatible version.

Hi It's correct and we are doing setup in this file.

Reply:
> What about just removing these lines?
> I find other tutorials don't have that kind of explicit package installation steps.
> But I don't know how they passed the CI.
> 
> If the `setup.py` is a must to pass the CI, please point an example (which I cannot find) to me.
> Thanks.

To see how this file is passed to CI, please refer to https://github.com/dmlc/gluon-nlp/blob/master/ci/prepare_clean_env.sh#L21. CI will use this shell file to prepare its conda environment and the L21 here installs necessary packages.